src/components/common/ProductBox/ProductBox.js

```diff
@@ -49,10 +51,28 @@ const ProductBox = ({
     <div className={styles.line}></div>
     <div className={styles.actions}>
       <div className={styles.outlines}>
-        <Button className={favourite ? styles.active : ''} variant='outline'>
+        <Button
+          onClick={
+            favourite
+              ? () => setFavouriteValue(id, false)
+              : () => setFavouriteValue(id, true)
+          }
+          className={favourite ? styles.active : ''}
+          variant='outline'
+          hrefAdress='javascript: void(0)'
+        >
           <FontAwesomeIcon icon={faHeart}>Favorite</FontAwesomeIcon>
         </Button>
-        <Button className={addToCompare ? styles.active : ''} variant='outline'>
+        <Button
+          onClick={
+            addToCompare
+              ? () => setAddToCompareValue(id, false)
+              : () => setAddToCompareValue(id, true)
+          }
+          className={addToCompare ? styles.active : ''}
+          variant='outline'
+          hrefAdress='javascript: void(0)'
```

Comment:
`preventDefault` zastosujmy. Ten trick jest przydatny jeśli nie mam JS który by działał, ale my nie dość że mamy JS to dodatkowo wykonujemy akcję na `onClick`